### ISSUE TITLE
Implement masking support

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#21]
Currently, none of the tf.layers.* Layers support masking.

**Comment by rthadur** [comment]
Closing the issue as the commit has been pushed , please feel free to reopen the issue if this has not been addressed.

**Comment by caisq** [comment]
This feature is not implemented yet.

**Comment by caisq** [comment]
Working on this issue now

**Comment by caisq** [comment]
Fixed already. See the date-conversion-attention example in tfjs-examples for usage example.
